client/webserver/api.go

```diff
@@ -168,21 +168,48 @@ func (s *WebServer) apiOpenWallet(w http.ResponseWriter, r *http.Request) {
 	if !readPost(w, r, form) {
 		return
 	}
-	has, on, _ := s.core.WalletStatus(form.AssetID)
-	if !has {
+	status := s.core.WalletState(form.AssetID)
+	if status == nil {
 		s.writeAPIError(w, "No wallet for %d -> %s", form.AssetID, unbip(form.AssetID))
 		return
 	}
-	if !on {
-		s.writeAPIError(w, "%s wallet not connected", unbip(form.AssetID))
-		return
-	}
 	err := s.core.OpenWallet(form.AssetID, form.Pass)
 	if err != nil {
 		s.writeAPIError(w, "error unlocking %s wallet: %v", unbip(form.AssetID), err)
 		return
 	}
-	s.notifyWalletUpdate(form.AssetID, true, true)
+	s.notifyWalletUpdate(form.AssetID)
+	writeJSON(w, simpleAck(), s.indent)
+}
+
+// apiOpenWallet conencts to a wallet backend.
```

Comment:
Could mention `/connectwallet` as the other  handler docs mention their associated API endpoint.